contents/docs/libraries/angular.md

```diff
@@ -192,7 +196,9 @@ export class PostHogSessionRecordingService {
   constructor(private ngZone: NgZone) {}
   startSessionRecording() {
     this.ngZone.runOutsideAngular(() => {
-      posthog.startSessionRecording()
+      posthog.init(
```

Comment:
should the method above be called `initPostHog` instead? which is what the code snippet does now and not only `posthog.startSessionRecording()`

Reply:
Ah, damn, yes, copy-pasta